src/Setup.elm

```diff
@@ -1,3 +1,17 @@
+<<<<<<< HEAD
```

Comment:
These shouldn't appear here.
It's here because you've missed these when you merged `upstream master` and commited the conflicts.